packages/react-components/src/localization/locales/ar-SA/strings.json

```diff
@@ -342,4 +342,4 @@
     "leftNavButtonAriaLabel": "الصفحة السابقة",
     "rightNavButtonAriaLabel": "الصفحة التالية"
   }
-}
\ No newline at end of file
+}

```

Comment:
It would be nice to leave these files unchanged if possible?

Reply:
This is something that happens in some PR's. I did not edit them but it is unknown why github will add or remove new lines from those files.

Reply:
This happens when we merge in from main. Prettier runs locally on the committed files, which is any file that changed in main since. This fixes the missing newline.
The problem is that the localization PRs spawned from the touchdown service ADO actions are not running prettier on themself.

Reply:
ahhh I see, yeah I did that to clear the conflicts